setup.py

```diff
@@ -34,6 +34,7 @@
     keywords="openqa opensuse fedora client",
     url="https://github.com/os-autoinst/openQA-python-client",
     packages=["openqa_client"],
+    package_data={"openqa_client": ["py.typed"]},
```

Comment:
Just to clarify, is this fixing CI checks that are currently broken? Or is there something we should be adding to make that happen? #49 doesn't mention how the errors were produced 🤔

Reply:
I guess that @ricardobranco777 was running mypy against a project that is using openQA-python-client. However, as we did not have this `py.typed` marker, mypy would not look into the sources to use the declared types for type checking, thereby producing the error message. By adding this file, the error should be gone

Reply:
You're right @dcermak